spec/features/airlines/updating_airline_spec.rb

```diff
@@ -29,7 +29,8 @@
 
       fill_in "Name", with: "Delta Airlines"
       fill_in "Link URL", with: "www.example.com"
-      fill_in "Logo URL", with: "http://img4.wikia.nocookie.net/__cb20110201093102/logopedia/images/f/f5/Logo1929.gif"
+      fill_in "Logo URL",
+       with: "http://img4.wikia.nocookie.net/__cb20110201093102/logopedia/images/f/f5/Logo1929.gif"
```

Comment:
Line is too long. [99/80]<br>Align the parameters of a method call if they span more than one line.